src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
handedness - what you suggest is not how the other controllers work.  you say what you want matched and the right tracked controls is injected if present.  same here, except you can leave hand blank meaning either should match.